templates/tfengine/components/devops/main.tf

```diff
@@ -120,31 +120,35 @@ resource "google_project_iam_binding" "devops_owners" {
   project = module.project.project_id
   role    = "roles/owner"
   {{- if get .project.owners_group "exists" false}}
```

Comment:
This can be replaced with missing_project_owners_group as well

Reply:
Done

Reply:
Done